content/docs/user-guide/programming/components/editor-components.md

```diff
@@ -1,6 +1,7 @@
 ---
-description: ' Learn about editor components in Open 3D Engine. '
-title: Editor Components
+linktitle: Editor Components
+title: Editor Components in O3DE
+description: Learn how to create editor components in Open 3D Engine.
 weight: 500
 ---
 
```

Comment:
In the following intro paragraph, change the first occurrence of "O3DE" to "**Open 3D Engine (O3DE)**".